.env.compose

```
@@ -230,6 +230,10 @@ GAUZY_OAUTH_APP_CLIENT_SECRET=
 GAUZY_OAUTH_APP_CODE_SECRET=
 GAUZY_OAUTH_APP_REDIRECT_URIS="http://localhost:4200/redirect,https://cloud.activepieces.com/redirect"
 
+#SIM platform integration
+SIM_DEFAULT_BASE_URL="https://www.sim.ai"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)